zest/releaser/git.py

```diff
@@ -32,12 +33,14 @@ def name(self):
         # No setup.py? With git we can probably only fall back to the directory
         # name as there's no svn-url with a usable name in it.
         dir_name = os.path.basename(os.getcwd())
+        if not isinstance(dir_name, six.text_type):
+            dir_name = dir_name.decode(sys.getfilesystemencoding())
```

Comment:
Maybe use `.decode(..., 'surrogateescape')` here [like Python 3 does](https://docs.python.org/3/library/os.html#file-names-command-line-arguments-and-environment-variables)?  (I expect it'll blow up later anyway, when things that expect valid Unicode strings encounter unpaired surrogates, but at least it'll blow up the same way on Python 2 and 3.)